scripts/skills/perks/perk_rf_phalanx.nut

```diff
@@ -114,13 +116,13 @@ this.perk_rf_phalanx <- ::inherit("scripts/skills/skill", {
 
 	function onQueryTooltip( _skill, _tooltip )
 	{
-		if (_skill.getID() == "effects.shieldwall" && this.hasAdjacentShieldwall())
+		if (_skill.getID() == "actives.shieldwall" && this.hasAdjacentShieldwall())
 		{
 			_tooltip.push({
 				id = 15,
 				type = "text",
 				icon = "ui/icons/special.png",
-				text = ::Reforged.Mod.Tooltips.parseString("Will not expire as you have [Phalanx|Perk+perk_rf_phalanx] and are next to an ally with [Shieldwall|Skill+shieldwall_effect]")
+				text = ::Reforged.Mod.Tooltips.parseString("Costs " + ::MSU.Text.colorizeMultWithText(this.m.ShieldWallActionPointsMult, {InvertColor = true}) + " [Action Points|Concept.ActionPoints] (to a minimum of " + ::MSU.Text.colorPositive(this.m.ShieldWallActionPointsMininum) + ") and " + ::MSU.Text.colorizeMultWithText(this.m.ShieldWallFatigueMult, {InvertColor = true}) + " [Fatigue|Concept.Fatigue]")
```

Comment:
This should probably indicate where the effect comes from (like the former text)